resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle-service/Dockerfile

```diff
@@ -15,25 +15,12 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
```

Comment:
imo there should be a tag per spark release, something like `spark-base:2.1.0`, etc

I'd propose renaming it `spark-kubernetes-base`